wp-content/mu-plugins/https.php

```diff
@@ -25,25 +26,27 @@ function wp_calculate_image_srcset( $sources ) {
 
 /**
  * Replace http:// with https:// in the embed code (before caching).
- * 
+ *
+ * @since  2.0.0
```

Comment:
Was going to ask about that, I thought it must have been something like that, but I got lazy 👅 .

As for the version, if you consider it in the right perspective you will find that the major is actually required. The names of the functions changed, and you are saying that they're only used here, but the fact is the "API" changed and it would break the usage of this plugin anywhere else.
I know it's a part of this repository, but one must not discard the chance of someone copy pasting this for their projects (😂  eheh). After all this is an isolated plugin :).

I considered changing the calls in the `add_action`s, in this case it would be a minor, but I think the names are more in line with the plugin's purpose. Still, there is time :)